news/2023/2023-11-07-winter-fanart-contest.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title: "Winter 2023 Fanart Contest"
+date: 2023-11-07 08:00:00 +0000
+---
+
+The end of the year soon approaches, and what better way to send it off in style than with a feast-themed fanart contest!
+
+![](https://assets.ppy.sh/contests/189/header.jpg)
+
+Each and every year since a date long-lost to the annals of time, we've sat down and posted the artists of the osu! community a timeless question: **do you like to draw?**
```

Comment:
```suggestion
Each and every year since a date long lost to the annals of time, we've sat down and posed the artists of the osu! community a timeless question: **do you like to draw?**
```

Reply:
libtex disagrees with you firmly re long-lost

Reply:
libtex is wrong because "long lost" isn't used to modify a noun, where it would be hyphenated

google "long lost to time". no one hyphenates "long lost" in this case